src/data_processing/netflix_to_wikidata.py

```diff
@@ -0,0 +1,110 @@
+import os
+from dotenv import load_dotenv
+import requests
+import pandas as pd
+
+load_dotenv()
+
+base_dir = os.getenv('BASE_DIR')
+user_agent = os.getenv('USER_AGENT')
+
+netflix_data_path = os.path.join(base_dir, 'netflix_movies.txt')
+output_csv_path = os.path.join(base_dir, 'netflix_to_wikidata.csv')
+
+netflix_data = pd.read_csv(netflix_data_path, delimiter=',', names=['NetflixId', 'Year', 'Title'],  encoding='ISO-8859-1', on_bad_lines='skip')
+
+def construct_query(title, year):
+    query = '''
+    SELECT * WHERE {
+        SERVICE wikibase:mwapi {
+            bd:serviceParam wikibase:api "EntitySearch" ;
+                            wikibase:endpoint "www.wikidata.org" ;
+                            mwapi:search "%s" ;
```

Comment:
for a query this big, I would name the params:

```
%(search_term)s
```

just so it's easier to understand in the future.